test/e2e/kuma_test.go

```diff
@@ -0,0 +1,151 @@
+//go:build e2e_tests
+// +build e2e_tests
+
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"github.com/blang/semver/v4"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/addons/kuma"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/addons/loadimage"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/addons/metallb"
+	"github.com/kong/kubernetes-testing-framework/pkg/environments"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestDeployAllInOneDBLESSKuma(t *testing.T) {
+	t.Log("configuring all-in-one-dbless.yaml manifest test")
```

Comment:
I'd suggest that we try to use variables/consts that contain the paths for those manifests instead of hardcoding them. (Hypothetical) Moving manifests around will result in a major debugging pain.

There's no need to do this in this PR (as an exception) but I believe this should be done at some point.